src/lib.rs

```diff
@@ -85,3 +85,6 @@ pub mod syscfg;
 pub mod time;
 pub mod timer;
 // pub mod watchdog;
+
+#[cfg(all(feature = "stm32-usbd", any(feature = "stm32g473",)))]
```

Comment:
The machine feature needs changed here. Do all G4's have USB?

Reply:
Per ST's website, yes they all have USB. I've been bringing a G431 up and this is the only line that needed adjusting. G431 is the minimum spec G4 available.